WooCommerce/Classes/ViewRelated/Orders/ShippingProvidersViewController.swift

```diff
@@ -0,0 +1,153 @@
+import UIKit
+import Yosemite
+
+protocol ShipmentProviderListDelegate: AnyObject {
+    func shipmentProviderList(_ list: ShippingProvidersViewController, didSelect: ShipmentTrackingProvider, groupName: String)
+}
+
+final class ShippingProvidersViewController: UIViewController {
+    private let viewModel: ShippingProvidersViewModel
+    private weak var delegate: ShipmentProviderListDelegate?
+
+    @IBOutlet weak var table: UITableView!
+
+    private lazy var searchController: UISearchController = {
+        let returnValue = UISearchController(searchResultsController: nil)
+        returnValue.hidesNavigationBarDuringPresentation = false
+        returnValue.dimsBackgroundDuringPresentation = false
+        returnValue.searchResultsUpdater = self
+        returnValue.delegate = self
+
+        returnValue.searchBar.tintColor = .black
+        returnValue.searchBar.backgroundColor = .white
+
+        return returnValue
+    }()
+
+    init(viewModel: ShippingProvidersViewModel, delegate: ShipmentProviderListDelegate) {
+        self.viewModel = viewModel
+        self.delegate = delegate
+        super.init(nibName: type(of: self).nibName, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigation()
+        configureSearchController()
```

Comment:
I suggest deleting the search bar methods until it's time to implement those. What do you think about implementing this part in a separate PR?